src/gc/gc.d

```diff
@@ -432,6 +484,8 @@ struct GC
                 p = sentinel_sub(p);
                 auto biti = cast(size_t)(p - pool.baseAddr) >> pool.shiftBy;
 
+                //Based on the documetnation, shouldn't we set the
+                //bits first and then get the new value of the bit field?
```

Comment:
Is it a good idea to include questions in the code? At least I find it super strange when I find questions like this in the code.